scripts/update-release.sh

```diff
@@ -0,0 +1,17 @@
+# Oh boy, this is a treat. Don't bother, just hope it never breaks. :D
```

Comment:
Maybe it's a little late but isn't this better to be a executable javascript file? Since most of the team can actually read that 🤔